src/main/kotlin/sidechain/iroha/consumer/IrohaNetworkImpl.kt

```diff
@@ -102,8 +102,18 @@ class IrohaNetworkImpl(host: String, port: Int) : IrohaNetwork {
     ): Result<List<String>, Exception> {
         send(batch)
 
-        val results = hashes.map { checkTransactionStatus(it) }.filter { it.component1() != null }.map { it.get() }
-        return Result.of { results }
+        return Result.of {
+            hashes.map { checkTransactionStatus(it) }
+                .filter {
+                    it.fold(
+                        { true },
+                        {
+                            logger.warn("Send batch: ", it)
```

Comment:
Can we add some details in this warning? "Send batch" doesn't sound like something bad happened.